### DISCUSSION TITLE
VITS model trained with VCTK with silence removed?

### DISCUSSION THREAD

**Discussion by skol101** [coqui-ai/TTS#824] [General Q&A] [answered]
Rearchers from SC-GlowTTS  https://github.com/Edresson/SC-GlowTTS used VCTK dataset with silence removed.

Have you trained vits---vctk model with or without silence? And does it have any effect on the quality of inference?

**Comment by loganhart02** [comment]
there is a pretrained vits model you can download. it’s in the models.json right under sc-glow-tts.

**Reply by skol101** [reply]
> there is a pretrained vits model you can download. it’s in the models.json right under sc-glow-tts.
> […](#)

I'm just curious what version of VCTK dataset was used: with a silence or without? And whether it has any implication.

**Comment by loganhart02** [comment, marked as answer]
I would say it’s with removed silences. as early versions of sc glow tts where trained on that looking back at the old configs. Also the original dataset has audio files with long periods of silence in them which can cause performance problems.
